$NetBSD: patch-aa,v 1.3 2017/09/09 20:12:49 dholland Exp $

Make sure we get iffe from its correct location.
Build for C89, not K&R C.

--- src/lib/sfio/Makefile.orig	1998-03-19 16:17:33.000000000 +0000
+++ src/lib/sfio/Makefile
@@ -7,7 +7,9 @@ VERSION = 1.0
 .INCLUDE : Stdio_b
 ancestor = 2
 
-CCFLAGS = -O
+CCFLAGS = -O -D__STD_C
+
+BINDIR=	../../../bin
 
 SFIO=	sfexit.c sfclose.c sfclrlock.c sfcvt.c sfdlen.c sfexcept.c sfextern.c \
 	sffilbuf.c sfflsbuf.c sfprints.c sfgetd.c sfgetl.c sfgetr.c sfgetu.c \
@@ -41,7 +43,7 @@ sfio  $(VERSION) :LIBRARY:	$(SFIO) $(SFI
 stdio $(VERSION) :LIBRARY:	$(STDIO_B)
 
 ast_common.h : features/common
-	iffe - run features/common > ast_common.h
+	${BINDIR}/iffe - run features/common > ast_common.h
 sfstdgen : sfstdio.c
 	$(CC) -I. $(*) -o sfstdgen
 sfstdhdr.h :
